tests/unit/bfs_fine_grained.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <gtest/internal/gtest-param-util-generated.h>
+#include <optional>
+#include <string>
+#include <unordered_map>
+#include "auth/models.hpp"
```

Comment:
empty line between. + gtest comes below of unordered map and above of auth models

Reply:
I fixed this but for some reason this is not outdated, you can see how I fixed it here https://github.com/memgraph/memgraph/blob/T0997-MG-authorization-on-paths/tests/unit/bfs_fine_grained.cpp